salesforce-connector-demo/src/salesforce/connector/demo/ChartBean.java

```diff
@@ -0,0 +1,154 @@
+import org.primefaces.model.charts.optionconfig.legend.LegendLabel;
+import org.primefaces.model.charts.optionconfig.title.Title;
+
+import com.axonivy.connector.salesforce.model.Opportunity;
+
+import ch.ivyteam.ivy.environment.Ivy;
+import salesforce.connector.demo.dto.OpportunityDTO;
+import salesforce.connector.demo.enums.Stage;
+
+public class ChartBean {
+	List<Opportunity> opportunities;
+	List<OpportunityDTO> opps;
+	private BarChartModel barModel;
+
+	public ChartBean() {
+		opps = new ArrayList<>();
+		opportunities = Utils.getAllOpps();
+		Ivy.log().info("list opportunities: " + opportunities.size());
+		
+		opps = Utils.convertToOppDTO(opportunities);
+		
+		createBarChartModel();
+	}
+		
+	public void createBarChartModel() {
+		barModel = new BarChartModel();
+        ChartData data = new ChartData();
+
+        BarChartDataSet dataSet = new BarChartDataSet();
+        dataSet.setLabel("Opportunities");
+
+        List<Number> values = new ArrayList<>();
+        values.add(countByStage(Stage.PROSPECTING.label));
+        values.add(countByStage(Stage.QUALIFICATION.label));
+        values.add(countByStage(Stage.NEEDS_ANALYSIS.label));
+        values.add(countByStage(Stage.VALUE_PROPOSITION.label));
+        values.add(countByStage(Stage.DECISION_MAKERS.label));
+        values.add(countByStage(Stage.PERCEPTION_ANALYSIS.label));
+        values.add(countByStage(Stage.PROPOSAL_PRICE_QUOTE.label));
+        values.add(countByStage(Stage.NEGOTIATION_REVIEW.label));
+        values.add(countByStage(Stage.CLOSED_WON.label));
+        values.add(countByStage(Stage.CLOSED_LOST.label));
+        dataSet.setData(values);
+
+        List<String> bgColors = new ArrayList<>();
+        bgColors.add("rgb(255, 99, 132)");
+        bgColors.add("rgb(75, 192, 192)");
+        bgColors.add("rgb(255, 205, 86)");
+        bgColors.add("rgb(201, 203, 207)");
+        bgColors.add("rgb(54, 162, 235)");
+        
+        bgColors.add("rgb(153, 102, 255)");
+        bgColors.add("rgb(153, 102, 51)");
+        bgColors.add("rgb(255, 51, 204)");
+        bgColors.add("rgb(204, 153, 0)");
+        bgColors.add("rgb(51, 153, 51)");
+        dataSet.setBackgroundColor(bgColors);
+
+        data.addChartDataSet(dataSet);
+        List<String> labels = new ArrayList<>();
+        labels.add(Stage.PROSPECTING.label);
+        labels.add(Stage.QUALIFICATION.label);
+        labels.add(Stage.NEEDS_ANALYSIS.label);
+        labels.add(Stage.VALUE_PROPOSITION.label);
+        labels.add(Stage.DECISION_MAKERS.label);
+        labels.add(Stage.PERCEPTION_ANALYSIS.label);
+        labels.add(Stage.PROPOSAL_PRICE_QUOTE.label);
+        labels.add(Stage.NEGOTIATION_REVIEW.label);
+        labels.add(Stage.CLOSED_WON.label);
+        labels.add(Stage.CLOSED_LOST.label);
+        data.setLabels(labels);
+
+        barModel.setData(data);
+        
+      //Options
+        BarChartOptions options = new BarChartOptions();
+//        options.setMaintainAspectRatio(false);
```

Comment:
Do we need to keep this line for example or can we remove it?
please check for line 96, @HoDacVu90